### DISCUSSION TITLE
Missing Metadata for Signac Analysis

### DISCUSSION THREAD

**Discussion by Jasminefighter** [stuart-lab/signac#1612] [Q&A]
Hi, 

I am trying to perform scATAC analysis from two published data - but neither has metadata. One paper has barcodes.tsv.gz, matrix.mtx.gz, peaks.bed.gz, and fragments.tsv.gz. The other paper has fragments.tsv.gz, fragments.tsv.gz.tbi.gz, and filtered_feature_bc_matrix.h5 files. 

Can I continue the analysis without running this step in the beginning?

metadata <- [read.csv](https://rdrr.io/r/utils/read.table.html)(
  file = "../vignette_data/atac_v1_pbmc_10k_singlecell.csv",
  header = TRUE,
  row.names = 1
)

 
Many Thanks!

**Comment by timoast** [comment]
That step loads the metadata generated by cellranger. You can continue without it, bear in mind that the cellranger-generated metadata will not be present in your object

**Comment by SmithTan01** [comment]
Hi,

I have a similar problem. I was wondering what the cell ranger-generated metadata file looked like? I believe these are the output files: https://www.10xgenomics.com/support/software/cell-ranger/latest/analysis/outputs/cr-outputs-gex-overviewm but I cannot see which one could be the metadata file?

I used cell ranger aggr for integration and the outputs are: https://www.10xgenomics.com/support/software/cell-ranger/latest/analysis/outputs/cr-outputs-aggr-outputs. However, again I cannot see which file could be the metadata.

Thank you for your help.
